colour/models/cie_xyy.py

```diff
@@ -134,6 +138,112 @@ def xyY_to_XYZ(xyY):
     return XYZ
 
 
+def xy_to_xyY(xy):
+    """
+    Converts from *xy* chromaticity coordinates to *CIE xyY* colourspace.
+
+    `xy` argument with last dimension being equal to 3 will be assumed to be a
```

Comment:
I was actually thinking about that when I wrote the comment, and thought uses could just multiply the appropriate index, but yeah lets do that, it will be much nicer.